eolearn/visualization/eoexecutor.py

```diff
@@ -69,8 +69,13 @@ def make_report(self, include_logs: bool = True) -> None:
             template = self._get_template()
 
             execution_log_filenames = [fs.path.basename(log_path) for log_path in self.eoexecutor.get_log_paths()]
-            if self.eoexecutor.save_logs:
-                execution_logs = self.eoexecutor.read_logs() if include_logs else None
+            if not include_logs:
+                execution_logs = None
+            elif self.eoexecutor.save_logs:
+                execution_logs = []
+                for log_path in self.eoexecutor.get_log_paths():
```

Comment:
There is now a for loop in place of the `ThreadPoolExecutor`. What are the implications of this w.r.t. the eo-grow pipeline executions?

Reply:
if there are a lot of them you shouldn't include logs anyway, or you get huge reports.
i can re-add the threadpool to try and improve efficiency a bit

Reply:
It wasn't a complaint, just an observation, since I'm not so familiar with the processes so deep down the hole

Reply:
it is a legitimate concern

Reply:
readded treaded loading